### ISSUE TITLE
Describe API in Swarm file

### ISSUE THREAD

**Issue by dev-tim** [WorldBrain/cortex#14]
I have a suggestion to use Swarm for API documentation.

**Comment by hoshomoh** [comment]
Since we are using Django rest framework for our API'S. I would suggest we use [Swagger](https://django-rest-swagger.readthedocs.io/en/latest/)

**Comment by dev-tim** [comment]
Oh, damn, sure I misspelled it :)

**Comment by oliversauter** [comment]
@hoshomoh When is the point where we need swagger in action? Can we finish up the prototype without having the api documentation?

**Comment by hoshomoh** [comment]
Definitely. So, we would just communicate how and what the API is expecting as parameter for now.
